### ISSUE TITLE
Payment details in order history

### ISSUE THREAD

**Issue by rehannali** [Shopify/mobile-buy-sdk-ios#944]
Hi, I search through forums and all places and read all documentation. I didn't find payment details in order history in sdk. I see this option on rest api but didn't find it in order history through sdk.

I'm using latest sdk version.
Please, could you guide me where or how i can find it or it is not implemented in sdk?

**Comment by dbart01** [comment]
That's correct, payment information is not currently available in customer orders. This is a limitation of Storefront API. I'll keep this issue open to track any future development.

**Comment by DrTeePot** [comment]
Hello @rehannali , what information specifically are you looking for?

**Comment by rehannali** [comment]
Hi, i need payment information for order through sdk from which order placed.

**Comment by morganmccunn** [comment]
Hi @rehannali - curious to understand what exactly this data would be used for. Can you please shed some light on how this would help the customer experience for your app?  Thanks!

**Comment by rehannali** [comment]
Actually, this was customer requirement to show details like last 4 digit from which they placed order to mentioned in order history to combine things so you know about details of order with payment method.

**Comment by userSV** [comment]
@rehannali Have you got any solution for this. I am also using the GraphQl to fetch the data of order but not able to get any payment information like payment method and last digits of card. It is working through rest api but i dont want to use the rest api as everything has been done through graphQL now. I dont understand why are there differences in data between sdk graphQl and the storefront apis.

**Comment by rehannali** [comment]
@userSV Currently, I'm not working on this product anymore so i can't give you exact answer to that question. As for REST APIs and GraphQL question, they built from scratch GraphQL that's why there are some limitations for this SDK.
